x/distribution/keeper/genesis.go

```diff
@@ -112,7 +112,17 @@ func (k Keeper) InitGenesis(ctx sdk.Context, data types.GenesisState) {
 		if err != nil {
 			panic(err)
 		}
-		err = k.SetValidatorSlashEvent(ctx, valAddr, evt.Height, evt.Period, evt.ValidatorSlashEvent)
+
+		err = k.ValidatorSlashEvents.Set(
+			ctx,
+			collections.Join3[sdk.ValAddress, uint64, uint64](
```

Comment:
you shouldn't need to type hint here, inference should work properly.

Reply:
ah didn't realize, ty!